OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -64,7 +64,7 @@ public void StartGame(Arguments args)
 			var connect = args != null ? args.GetValue("Launch.Connect", null) : null;
 			if (!string.IsNullOrEmpty(connect))
 			{
-				var parts = connect.Split(':');
+				var parts = connect.Replace("openra://", "").Split(':');
```

Comment:
Got it to work again in revision 11 of https://build.opensuse.org/package/show/games:openra/development.